### DISCUSSION TITLE
DreamBooth update screw the process

### DISCUSSION THREAD

**Discussion by meunumerotim333** [TheLastBen/fast-stable-diffusion#1143] [Q&A] [answered]
DreamBooth isn't adding the trained model to the model, is writing and erasing the existed. My english isn't fluent.. all I know is the last 3 trained model I did today doesn't work. The prompt words of model simple don't activate like "SamDoesArts style" from samdoesarts model.. etc

**Comment by theridon** [comment, marked as answer]
Try to delete your SD folder and run the collab again, that fixed it f or me

**Reply by meunumerotim333** [reply]
Did that, it slight work with nitro_diffusion model as arcane style but didn't work with modern disney style.. so still bad. Also tried with SamDoesArts again and isn't working (using ultmate merge version). I'm using the old copy of DreamBooth and its working. But this new one isn't.
